objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
> `[['Bernat', 32, 'Barcelona'], ['Joan', 41, 'Barcelona']]`

My point exactly! This would **obviously** map to CSV as:

    "Bernat","32","Barcelona"
    "Joan","41","Barcelona"

Nobody would ever write it it as:

    "Bernat","Joan"
    "32","41"
    "Barcelona","Barcelona"

Whereas if we're using a list to store a single magnitude, like:

    names = ["Bernat", "Joan", "Michael"]

Then, of course, it makes sense to "columnize" them. (Or not! Who cares!)

But this discussion is already pointless since [I've updated my PR](https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/pull/1670/commits/678d65bca9788341a63f3536053a3032d31ae30a) so that it splits by newlines.

By the way, Scratch 1.4 did export lists as newline-separated values, but *also* as comma-separated values in their HTTP API. What's most useful depends on what your lists represent and what you're going to do with the data after exporting it.

Can we please **\</discussion>** now? :)